curriculum/challenges/english/25-front-end-development/workshop-mathbot/66ea7fc048a6e26e33c598fe.md

```diff
@@ -0,0 +1,61 @@
+---
+id: 66ea7fc048a6e26e33c598fe
+title: Step 3
+challengeType: 1
+dashedName: step-3
+---
+
+# --description--
+
+Now, it is time to generate a random number using the `Math.random()` method.
+
+Create a variable called `randomNum` and assign it the value of the result of calling the `Math.random()` method.
+
+Then, log the `randomNum` variable to the console.
+
+Try refreshing the page a few times to see the different random numbers that are generated.
```

Comment:
I'm fine with it, but maybe a different suggestion other than refreshing the page. Maybe "Try adding a space in the code to re-run the bot and see different random numbers that are generated.